app/package.json

```diff
@@ -8,24 +8,24 @@
     "@fuel-ui/css": "^0.23",
     "@fuel-ui/react": "^0.23.3",
     "@fuels/connectors": "0.5.0",
-    "@fuels/react": "0.20.0",
+    "@fuels/react": "0.25.0",
     "@mui/base": "^5.0.0-beta.2",
     "@mui/icons-material": "^5.11.16",
     "@mui/lab": "^5.0.0-alpha.46",
     "@mui/material": "^5.13.2",
-    "@tanstack/react-query": "^5.29.0",
+    "@tanstack/react-query": "5.35.1",
     "@vercel/analytics": "^1.2.2",
     "ace-builds": "^1.22.0",
     "ace-mode-solidity": "^0.1.1",
     "ansicolor": "^1.1.100",
     "await-timeout": "^1.1.1",
-    "fuels": "0.88.1",
+    "fuels": "0.94.0",
```

Comment:
We just released [v0.94.1](https://github.com/FuelLabs/fuels-ts/releases/tag/v0.94.1), containing minor fixes and touch-ups.

They don't seem necessary to this PR, but since we're here, you could update it.